### PR TITLE
Update title, description, and icon of Post Categories

### PR DESCRIPTION
**Pull request by carolinan**
## Description

Update title, description and icon of the Post Categories block variation, one possible or temporary solution for #29399

## How has this been tested?

1. Activated a theme with support for full site editing
2. Added the post categories block,
3. Viewed the icon and information in the block settings sidebar.

## Screenshots 
The block title, description and icon in the sidebar displays the information about the block variation

## Types of changes

Enhancement

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Review by ntsekouras — COMMENTED** [review]
Hey 👋 - thanks for looking into this!

After https://github.com/WordPress/gutenberg/pull/27469, the only changes that are needed are in the `block variation`:
1. Add the description you want
2. Add `isActive` prop

```
isActive: ( blockAttributes ) => blockAttributes.term === 'category',
```

This will propagate the proper information in other places are well (BlockSwitcher, breadcrumbs etc...).

**Comment by carolinan** [comment]
I changed the code to isActive.

Should the current dashicon be used, or the svg category icon? The svg is used by the categories block:

image

**Review by ntsekouras — COMMENTED** [review]
Both lines need to follow GB lint rules. Other than that it works as expected. Thanks!
Make the changes and we'll 🚢 .

> Should the current dashicon be used, or the svg category icon? The svg is used by the categories block:

Current PR can be merged with the previous icon that was used. Having said that @jasmussen can chime in to share his thoughts.

**Comment by jasmussen** [comment]
It would be nice to have a new icon for post categories, that shared DNA with categories but was unique. I'll see if I can draw one, but I don't think it should block this PR.

File: packages/block-library/src/post-hierarchical-terms/variations.js [hunk 1]
```diff
@@ -7,11 +7,11 @@ const variations = [
 	{
 		name: 'category',
 		title: __( 'Post Categories' ),
-		description: __("Display a post's categories."),
+		description: __( "Display a post's categories." ),
```
Comment (jasmussen) [inline packages/block-library/src/post-hierarchical-terms/variations.js:10]: While we're here, should we change this to something less fiddly, such as "Display the categories of a post."?

**Review by ntsekouras — APPROVED** [review]
Looks good 👍 - Thanks!

File: packages/block-library/src/post-hierarchical-terms/variations.js (hunk 1 above)
Comment (ntsekouras) [reply packages/block-library/src/post-hierarchical-terms/variations.js:10]: I think we can land now and handle this in combination with an icon change, what do you think?

Comment (jasmussen) [reply packages/block-library/src/post-hierarchical-terms/variations.js:10]: Sure.

**Comment by jasmussen** [comment]
Created https://github.com/WordPress/gutenberg/pull/29553.

**Pull request merged**